SalesforceCommerceCloud/pwa-kit
# Improve error pages 404

This PR improves the 404 page to match the look and feel of the pwa kit based on the [design](https://www.figma.com/file/gOUKasMrK16zS5qrLCTSJl/PWA-Design-Kit?node-id=887%3A110731)


**JIRA**: https://gus.lightning.force.com/a07AH000000R2voYAC
 **Linked PRs**: (links to corresponding PRs, optional)

## How to test-drive this PR
- npm start
- navigate to a un-routed route: `localhost:3000/abc`
- You should see the 404 page is styled as per design


## Changes
- (change1)

## Todos

### General

### Backwards Compatibility

> Breaking changes include:
> * Removing a public function/Component/Prop
> * Adding required arguments to functions
> * Changing the data-types of function params or return values

### Accessibility Compliance
You must check off all items in **one** of the follow two lists:

_or..._


### Documentation


> Documentation and comments should follow the [Mobify Content Style Guide](https://docs.google.com/a/mobify.me/document/d/1jlcg5boC3MUHN7fy2n3Yu_FBGzGgTUPrnMoePzQtqE4/edit?usp=sharing) and [Mobify's Capitalization and Spelling List](https://docs.google.com/document/d/1LO7RAr2vD3LFs_bj5j0vIFMKsVFrOFfEjpCheYEH3Lg/edit?usp=sharing)
